app/scripts/modules/amazon/serverGroup/details/serverGroupDetails.aws.controller.js

```diff
@@ -341,6 +341,13 @@ module.exports = angular.module('spinnaker.serverGroup.details.aws.controller',
             var cluster = _.find(app.clusters, {name: this.serverGroup.cluster, account: this.serverGroup.account});
             return _.filter(cluster.serverGroups, {isDisabled: true, region: this.serverGroup.region});
           },
+          allServerGroups: () => {
+            var cluster = _.find(app.clusters, {name: this.serverGroup.cluster, account: this.serverGroup.account});
+            var selectedServerGroup = this.serverGroup;
+            return _.filter(cluster.serverGroups, function (serverGroup) {
+              return serverGroup.name !== selectedServerGroup.name && serverGroup.region === selectedServerGroup.region;
+            });
```

Comment:
I think this can be simplified:
```javascript
allServerGroups: () => app.getDataSource('serverGroups').data
  .filter(g => g.cluster === this.serverGroup.cluster && 
    g.region === this.serverGroup.region && 
    this.account === this.serverGroup.account && 
    g.name !== this.serverGroup.name);
```